ipa-core/src/protocol/step/steps.txt

```diff
@@ -7978,6 +7978,386 @@ ipa_core::protocol::ipa_prf::Step::convert_input_rows_to_prf/ipa_core::protocol:
+ipa_core::protocol::ipa_prf::Step::sort_by_timestamp/ipa_core::protocol::ipa_prf::quicksort::Step::quicksort_pass10
+ipa_core::protocol::ipa_prf::Step::sort_by_timestamp/ipa_core::protocol::ipa_prf::quicksort::Step::quicksort_pass10/ipa_core::protocol::ipa_prf::quicksort::Step::compare
+ipa_core::protocol::ipa_prf::Step::sort_by_timestamp/ipa_core::protocol::ipa_prf::quicksort::Step::quicksort_pass10/ipa_core::protocol::ipa_prf::quicksort::Step::compare/ipa_core::protocol::step::BitOpStep::bit0
+ipa_core::protocol::ipa_prf::Step::sort_by_timestamp/ipa_core::protocol::ipa_prf::quicksort::Step::quicksort_pass10/ipa_core::protocol::ipa_prf::quicksort::Step::compare/ipa_core::protocol::step::BitOpStep::bit1
+ipa_core::protocol::ipa_prf::Step::sort_by_timestamp/ipa_core::protocol::ipa_prf::quicksort::Step::quicksort_pass10/ipa_core::protocol::ipa_prf::quicksort::Step::compare/ipa_core::protocol::step::BitOpStep::bit10
+ipa_core::protocol::ipa_prf::Step::sort_by_timestamp/ipa_core::protocol::ipa_prf::quicksort::Step::quicksort_pass10/ipa_core::protocol::ipa_prf::quicksort::Step::compare/ipa_core::protocol::step::BitOpStep::bit11
+ipa_core::protocol::ipa_prf::Step::sort_by_timestamp/ipa_core::protocol::ipa_prf::quicksort::Step::quicksort_pass10/ipa_core::protocol::ipa_prf::quicksort::Step::compare/ipa_core::protocol::step::BitOpStep::bit12
+ipa_core::protocol::ipa_prf::Step::sort_by_timestamp/ipa_core::protocol::ipa_prf::quicksort::Step::quicksort_pass10/ipa_core::protocol::ipa_prf::quicksort::Step::compare/ipa_core::protocol::step::BitOpStep::bit13
+ipa_core::protocol::ipa_prf::Step::sort_by_timestamp/ipa_core::protocol::ipa_prf::quicksort::Step::quicksort_pass10/ipa_core::protocol::ipa_prf::quicksort::Step::compare/ipa_core::protocol::step::BitOpStep::bit14
+ipa_core::protocol::ipa_prf::Step::sort_by_timestamp/ipa_core::protocol::ipa_prf::quicksort::Step::quicksort_pass10/ipa_core::protocol::ipa_prf::quicksort::Step::compare/ipa_core::protocol::step::BitOpStep::bit15
+ipa_core::protocol::ipa_prf::Step::sort_by_timestamp/ipa_core::protocol::ipa_prf::quicksort::Step::quicksort_pass10/ipa_core::protocol::ipa_prf::quicksort::Step::compare/ipa_core::protocol::step::BitOpStep::bit16
+ipa_core::protocol::ipa_prf::Step::sort_by_timestamp/ipa_core::protocol::ipa_prf::quicksort::Step::quicksort_pass10/ipa_core::protocol::ipa_prf::quicksort::Step::compare/ipa_core::protocol::step::BitOpStep::bit17
+ipa_core::protocol::ipa_prf::Step::sort_by_timestamp/ipa_core::protocol::ipa_prf::quicksort::Step::quicksort_pass10/ipa_core::protocol::ipa_prf::quicksort::Step::compare/ipa_core::protocol::step::BitOpStep::bit18
+ipa_core::protocol::ipa_prf::Step::sort_by_timestamp/ipa_core::protocol::ipa_prf::quicksort::Step::quicksort_pass10/ipa_core::protocol::ipa_prf::quicksort::Step::compare/ipa_core::protocol::step::BitOpStep::bit19
```

Comment:
I don't understand why there are only 20 steps for comparison. That seems like it's only comparing BA20s, but our sort key is bigger now right? Shouldn't there be 32?